test/test_readExcel.py

```diff
@@ -7,7 +7,7 @@
 
 
 def test_Init_haifa():
-    file_path = os.path.dirname(__file__)+"\\..\\src\\config.json"
+    file_path = os.path.dirname(__file__)+"/../src/config.json"
```

Comment:
I looked and instead of doing this there is a function that called os.path.join
which solve this problem agnostic to Operation system.
But this is good correction